curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9e769df38c92635c158ba.md

```diff
@@ -258,7 +258,7 @@ function calculateCalories(e) {
   const remainingCalories = budgetCalories - consumedCalories + exerciseCalories;
   const surplusOrDeficit = remainingCalories < 0 ? 'Surplus' : 'Deficit';
   output.innerHTML = `
-  <span class="${surplusOrDeficit.toLowerCase()}">${remainingCalories} Calorie ${surplusOrDeficit}</span>
+  <span class="${surplusOrDeficit.toLowerCase()}">${Math.abs(remainingCalories)} Calorie ${surplusOrDeficit}</span>
```

Comment:
You should not be making changes to the seed code here.
Please revert this back to the way it was.

The issue should be to update the description.

Please re read through this issue again to make the correct changes 
https://github.com/freeCodeCamp/freeCodeCamp/issues/53487

Reply:
I am Sorry, I sincerely apologize for any inconvenience caused by my contribution.

Reply:
@incruder1 

You don't have to apologize. 
Your are new to contributing. 
It is all part of the learning process. 

When contributing, you want to make sure to read through the issue carefully to understand what changes should be made.  👍